qemu/tests/cfg/numa_memdev_options.cfg

```diff
@@ -39,5 +39,15 @@
         - numa_hugepage:
             backend_mem = memory-backend-file
             setup_hugepages = yes
-            mem-path = /mnt/kvm_hugepage
-            target_hugepages = 1500
+            mem-path_mem0 = /mnt/kvm_hugepage
+            mem-path_mem1 = /mnt/kvm_hugepage
+            target_nodes = 0 1
+            target_num_node0 = 522
+            target_num_node1 = 1546
```

Comment:
This value is based on the `size_mem0`, `size_mem1`,  and for 2M hugepage.
Add a comment to explain it is necessary,  when we run 1G or other hugepage, we need modify it accordingly.

Reply:
@yanan-fu Thanks, updated.